### PR TITLE
added react-lazy-load on home page

### PR DESCRIPTION
**Pull request by crater90**
# Description

Added Lazy load for individual sections on home page. Though i have lazy loaded only Tweets and Contributors sections because only these sections are dealing with a lot of media and network requests. All the other sections are small, static and aren't sharing resources over network.

Added the react-lazy-load library. 
Used the Lazyload component provided by library to load the Tweets and Contributors section. 

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Checked the network requests in Network tab while scrolling.

Steps to reproduce : 
1. Open the Network tab and reload the page.
2. Before implementation all the resources related to tweets and contributors gets fetched initially.
3. After implementation resources related to tweets starts getting fetched as the user scrolls to tweet section with an offset of 100px and contributors section with an offset of 50px

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by crater90** [comment]
@koustov @atapas can you please review the code changes...

**Comment by atapas** [comment]
> @koustov @atapas can you please review the code changes...

Yes will do.

**Comment by koustov** [comment]
@crater90 what is the issue this PR is targeting for?

**Comment by murtuzaalisurti** [comment]
> @crater90 what is the issue this PR is targeting for?

it's #463 I guess @crater90

File: src/common/home/Home.jsx [hunk 1]
```diff
@@ -111,6 +112,7 @@ const Home = () => {
           </Link>
         </div>
       </section>
+      <LazyLoad offset={100}>
```
Comment (murtuzaalisurti) [inline src/common/home/Home.jsx:115]: what's the reason for this specific offset?

**Review by murtuzaalisurti — CHANGES_REQUESTED** [review]
looks fine

**Pull request closed without merging**